src/control.lua

```diff
@@ -22,16 +22,15 @@ end)
 
 script.on_event("todolist-toggle-ui", function(event)
     local player = game.players[event.player_index]
-    if todo.get_main_frame(player) then
-        todo.minimize(player)
-    else
-        todo.maximize(player)
-        todo.refresh_task_table(player)
-    end
+    todo.toggle_ui(player)
 end)
 
 script.on_event(defines.events.on_runtime_mod_setting_changed, function(event)
     local player = game.players[event.player_index]
     local key = event.setting
     todo.on_runtime_mod_setting_changed(player, key)
-end)
\ No newline at end of file
+end)
+
+require("faketorio.core")
+
+require("faketorio.features.UI_feature")
```

Comment:
I don't know why this is missing on master now, I think it was in master when I started this branch?  Seems to be important for running faketorio tests ¯\_(ツ)_/¯

Reply:
This should not be in master. Because if it is there then you can run faketorio tests, but it also requires everyone to have faketorio installed. One of the features of faketorio is to generate these lines for you when you run the `test` command :)

Reply:
As already said in the main thread: This should not be here otherwise it won't work without faketorio anymore. Please remove it :D

Reply:
Ah! that explains why the lines suddenly appeared. Thanks, will do.

Reply:
Actually, it does work without faketorio (at least on windows).  I'm impatient, so I packaged it all up and put the .zip on my windows gaming machine (where I don't have faketorio installed) and it worked great for a few hours last night.  Just FYI - still planning to remove it.